src/components/pages/home/FeaturedCaseStudy.tsx

```diff
@@ -16,16 +16,39 @@ const TextContainer = styled.div`
     width: calc(50% - ${GUTTER_SHIFT});
   }
 `;
+// The gradient will overlap the "Built for Results" component, on desktop, until padding is added between each sections
+const Gradient = styled.div`
+  background: linear-gradient(
+    180deg,
+    rgba(6, 2, 29, 1) 0%,
+    rgb(222 56 86) 11%,
+    rgba(45, 45, 120, 1) 80%,
+    rgba(22, 21, 69, 1) 88%,
+    rgba(4, 0, 27, 1) 91%
+  );
+  bottom: 0;
+  height: 123%;
+  left: -1.5rem;
+  position: absolute;
+  top: -4rem;
+  width: calc(100% + (2 * 1.5rem));
```

Comment:
🔧 There's a handful of places in this `Gradient` that have spacing values that match the `spacing.ts` file. Consider leveraging the theming in those cases.

e.g. (note that you'll need to `import { spacing } from '~/theme/spacing';` for this usage.

```suggestion
  width: calc(100% + (2 * ${spacing.sm}));
```